feature-libs/product-configurator/common/assets/translations/en/configurator-common.ts

```diff
@@ -11,6 +11,9 @@ export const configurator = {
       updateMessage: 'The configuration is being updated in the background',
       showMore: 'show more',
       showLess: 'show less',
+      items: '{{param}} items',
```

Comment:
What if we only have one item?

Reply:
It was discussed with our PO and decided to have messages in plural.